CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 April 12th, 2018
 
+This minor release of the Optimizely SDK on 2.0.0 which is a major release.
+
```

Comment:
It's a patch release =) 

Should also update the version to 2.0.1

Reply:
oops.  getting ahead of myself here.